analysis_options.yaml

```diff
@@ -168,7 +168,7 @@ linter:
     - prefer_initializing_formals
     - prefer_inlined_adds
     # - prefer_int_literals # not yet tested
```

Comment:
while you're at it you could say that `prefer_int_literals # conflicts with https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#use-double-literals-for-double-constants`